ols/src/query_helpers/question_validator.py

```diff
@@ -52,7 +52,9 @@ def validate_question(
         logger.info(f"{conversation} Validating query")
 
         bare_llm = LLMLoader(
-            self.provider, self.model, params={"min_new_tokens": 1, "max_new_tokens": 4}
+            self.provider,
+            self.model,
+            params={"min_new_tokens": 1, "max_new_tokens": 12},
```

Comment:
does this work for both BAM and OpenAI?  At one point it worked for one but not the other

Reply:
Yeah it was chopping off the CATEGORIZATION value. Increasing it to 12 worked for both. Since I have removed categorization now, I have reverted it back to 4